### ISSUE TITLE
Strange behavior with MDT and HA

### ISSUE THREAD

**Issue by Trefex** [XKNX/xknx#527]
**Description of problem:**

- [x] using Home-Assistant knx plugin

**Version information:**
- xknx / Home-Assistant release with the issue: 0.116.4

```
  climate:
    - name: 'Office'
      temperature_address: '2/4/0'
      target_temperature_address: '2/4/1'
      target_temperature_state_address: '2/4/3'
      temperature_step: 0.5
      min_temp: 17.0
      max_temp: 25.0
```

The problem is that in HA, if I change the temperature, it changes `target_temperature_address` as it should and `target_temperature_state_address` is also updated correctly.

However, on my VisuControl Easy app, the mechanism is to change just the setpoint but not the setpoint comfort value as well. (see picture of resulting state in the KNX system)
Unfortunately, it uses a 1-bit DPT so I can't use the setpoint_shift stuff.

![image](https://user-images.githubusercontent.com/2877665/101998760-5b751600-3cd6-11eb-86f3-7584139f2a7d.png)

So when I use this app, the setpoint value in HA flickers to the newly set value (by the Visu app) and changes back to the value of `target_temperature_address` instead of requesting a `GroupValueRead` on `target_temperature_state_address`. 

Now comes the funny part, if I actually request a `GroupValueRead` on `target_temperature_state_address` via ETS, the value in HA is immediately updated and reflects the proper value.

What am I missing here?

Thanks !

**Comment by farmio** [comment]
> Unfortunately, it uses a 1-bit DPT so I can't use the setpoint_shift stuff.
> What am I missing here?

setpoint_shift with DPT1 is simply not supported. Use another temperature set mode if you want to use the MDT app along with HA.

**Comment by Trefex** [comment]
But it shouldn’t matter the way that the set point is set? 

Why is HA using the set point value from another GA? As soon as the new set point value is sent on the Bus, HA should take that value, instead of whatever it takes? 

I edited heavily my initial post. Could you please check how I formulated it ?

**Comment by Trefex** [comment]
@farmio can you explain to me in ELI5 why the fact that the app uses 1 but DPT, HA is unable to read the proper value from the GA it’s mapped to ? 

I’m trying to understand as it doesn’t make any sense currently to me.

**Comment by Trefex** [comment]
It turns out that target_temperature_state_address and target_temperature_address should be the same value.

I switched the Visu app to change the comfort setpoint instead of using 1 Byte change.

It works now as intended.
